packages/@okta/vuepress-site/docs/guides/authenticators-okta-email/main/aspnet/integratechallengemagiclink.md

```diff
@@ -141,7 +141,7 @@ For example, if the user has previously enrolled the email authenticator, Google
 
```

Comment:
Feedback on the new diagram:

1. In Step 4, "Returns status AwaitingChallenge..." should be "Return status AwaitingChallenge..."
2. In Step 4, make title consistent with text. Is it "4. Display a list of available ..." or 4. Display a list of possible ..."
3. In Step 6, the response arrows are pointing the wrong way.
4. Send email lto user's email address doesn't have a corresponding step in the content. Either add an item in the content or remove it from the diagram.
5. "7. Click the email magic link" text is not consistent with the corresponding header content
6. "8. Handle redirect in your app" text is not consistent with the corresponding header content
7. "Step 9. Complete challenge and sign user in" is missing from the content. Either remove it from the diagram or add it as a step in the content.
8. In step 8, "Passing in ..." should be "passing in ..."
9. In step 8, why is OPT bold in "Validate OTP". To be consistent with bolding the verbs just Validate should be bolded.

Reply:
I have fixed all issues.

Reply:
For step 8 "Passing in" should be "passing in" since it's the same sentence.